src/components/capture/create/SpecForm.tsx

```diff
@@ -53,16 +53,6 @@ function NewCaptureSpecForm(props: NewCaptureSpecFormProps) {
         }
     }, [discovered_catalog, setLink]);
 
-    const uiSchema = useMemo(() => {
-        let response = null;
-
-        if (endpointSchema.type) {
-            response = generateCustomUISchema(endpointSchema);
-        }
-
-        return response;
-    }, [endpointSchema]);
-
```

Comment:
This causes a weird issue with the collapsible group not having the data it needs. Still not fully sure why but we can figure that out in the future.